web-dev/src/contexts/auth-context.js

```diff
@@ -43,9 +43,9 @@ export const AuthProvider = ({children}) => {
   /**
    * Sign out of the service.
    */
-  const signOut = async () => {
+  const signOut = () => {
     // Invalidate current user data
-    await doLogout().finally(() => setUserData(null));
+    return doLogout().finally(() => void setUserData(null));
```

Comment:
Why is it void setUserData(null)?

Reply:
To tell the arrow function that the return of `setUserData()` is not intended to be used. [For more info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void#non-leaking_arrow_functions)